source/acquaman/IDEAS/IDEASXASScanConfiguration.h

```diff
@@ -31,7 +31,6 @@ class IDEASXASScanConfiguration : public AMStepScanConfiguration
 	Q_PROPERTY(QString edge READ edge WRITE setEdge)
 	Q_PROPERTY(double edgeEnergy READ energy WRITE setEnergy)
 	Q_PROPERTY(bool useFixedTime READ useFixedTime WRITE setUseFixedTime)
-	Q_PROPERTY(int numberOfScans READ numberOfScans WRITE setNumberOfScans)
 
```

Comment:
It does affect the database, but not in a way that's necessarily bad.  What this will do is "orphan" that column in the database.  The proper "robust and complete" solution would actually involve a database upgrade where the column is deleted from the necessary tables, but that is not necessary in this case.